src/main/java/com/alkemy/ong/web/controllers/GlobalExceptionHandler.java

```diff
@@ -17,21 +17,27 @@
 @ControllerAdvice
 public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
 
-        @ExceptionHandler(value
-                = { ResourceNotFoundException.class, ResourceNotFoundException.class })
-        protected ResponseEntity<Object> handleConflict(
-                RuntimeException ex, WebRequest request) {
-            return handleExceptionInternal(ex, ex.getMessage(),
-                    new HttpHeaders(), HttpStatus.NOT_FOUND, request);
-        }
+    @ExceptionHandler(ResourceNotFoundException.class)
+    protected ResponseEntity<Object> handleConflict(
+            RuntimeException ex, WebRequest request) {
+        return handleExceptionInternal(ex, ex.getMessage(),
+                new HttpHeaders(), HttpStatus.NOT_FOUND, request);
+    }
 
     @Override
     protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex, HttpHeaders headers, HttpStatus status, WebRequest request) {
         List<String> message = ex.getBindingResult()
                 .getAllErrors()
                 .stream()
-                .map(m->m.getDefaultMessage())
+                .map(m -> m.getDefaultMessage())
                 .collect(toList());
         return new ResponseEntity(message, HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(NullPointerException.class)
+    protected ResponseEntity<Object> handleComparatorConflict(
+            RuntimeException ex, WebRequest request) {
+        return handleExceptionInternal(ex, "PathId does not match with RequestRody ID.",
+                new HttpHeaders(), HttpStatus.NOT_FOUND, request);
```

Comment:
q este caso sea Bad request, ya q es un error del cliente.
El not found es exclusivo para cuando no se encuentra un recurso